pkg/jobcreator/controller.go

```diff
@@ -190,6 +220,56 @@ func (controller *JobCreatorController) Start(ctx context.Context, cm *system.Cl
 
 	return errorChan
 }
+func (controller *JobCreatorController) UpdateModuleAllowlist() error {
+	// URL of the GitHub raw content of the allowlist
+	allowlistUrl := "https://raw.githubusercontent.com/lilypad-tech/module-allowlist/master/allowlist.txt"
```

Comment:
Could we make this a package level const? If we won't change it at runtime, we could have it as a read-only value.

Also, is this URL correct? Currently reports a 404.

Reply:
Makes way more sense than a hardcoded string! The allowlistURL is now sourced from a package-level constant instead of being hard-coded within the function. The module-allowlist is now configured on Github so you should not be seeing a 404 error any longer.